src/crimson/os/seastore/cache.h

```diff
@@ -476,6 +476,8 @@ class Cache : public ExtentTransViewRetriever {
       ext_type);
 
     CachedExtent* p_extent;
+    bool needs_step_2 = false;
+    bool needs_touch = false;
     if (extent->is_stable()) {
```

Comment:
After https://github.com/ceph/ceph/pull/63218#discussion_r2088165738, and starting from this case, it should be possible to see the extent is a retired-placeholder.